ui.apps/src/main/content/jcr_root/apps/etoolbox-query-kit/components/console/queryStatistic/clientlibs/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let us move to the main clientlibs folder. The rest of css/js in one place. I frankly see no big reason fo these two rules to be in another